packages/framework/src/Form/Admin/PromoCode/PromoCodeLimitType.php

```diff
@@ -47,18 +47,18 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $options = $options['discount'];
 
         foreach ($options['constraints'] as $constraint) {
-            $constraint->groups = [PromoCodeFormTypeExtension::VALIDATION_GROUP_TYPE_PERCENT];
```

Comment:
JS files for promo codes are still in project-base. Is it intentional?

Reply:
ah, ehm, moved along with some other things :smile: 32ec0aea24